src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/Properties/Resources.Designer.cs

```diff
@@ -1,10 +1,10 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
-//     This code was generated by a tool.
-//     Runtime Version:4.0.30319.42000
+//     此代码由工具生成。
+//     运行时版本:4.0.30319.42000
 //
-//     Changes to this file may cause incorrect behavior and will be lost if
-//     the code is regenerated.
```

Comment:
This file changes comments from English. Can you please restore these comments to their original state?